app/src/engineering/java/com/hedvig/app/MockContractDetailViewModel.kt

```diff
@@ -32,7 +32,7 @@ class MockContractDetailViewModel : ContractDetailViewModel() {
                     DocumentItems.Document(
                         titleRes = R.string.MY_DOCUMENTS_INSURANCE_TERMS,
                         subTitleRes = R.string.insurance_details_view_documents_insurance_letter_subtitle,
-                        uri = Uri.parse(it.termsAndConditions.url)
+                        uri = Uri.parse(it.termsAndConditions?.url)
```

Comment:
These changes should not be required. Someone has snuck in a breaking change into the parent schema, and they will have to roll that breaking change back

Reply:
Oopsies, I just wanted it to compile for now, will revert it back when it's rolled back then